docs/02-guides/translate-your-application.mdx

```diff
@@ -0,0 +1,303 @@
+});
+```
+
+```mdx-code-block
+</TabItem>
+</Tabs>
+```
+
+## Translate what's in your components
+
+Now that you have defined what should be translated in your application, you
+actually need to translate your application. This is a two-step process:
+
+1. Run the following script that will fetch all your translatable strings in
+   your application and gather them in a JSON file located in
+   `translations/[locale].json`
+
+   ```shell
+   $ front-commerce translate <path> --locale <locale>
+   ```
+
+   :::tip ProTip™
+
+   The script has already been defined in the
+   [skeleton template](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/skeleton/package.json?ref_type=heads#L13),
+   and can be used directly with `pnpm run translate`. You can change the
+   default locale to accommodate your store.
+
+   :::
+
+2. Translate the strings available in those files, and you are good to go 🙂
+
+   :::tip ProTip™
+
+   You can use tools like [BabelEdit](https://www.codeandweb.com/babeledit) to
+   help translate and keep your translations in sync.
+
+   :::
+
+### Translations fallback
+
+With `react-intl` translations are usually grouped into a single file. In our
+case, we would expect them to be in `translations/[locale].json`. But in
+Front-Commerce's case, we don't want you to be troubled by translations that are
+handled by the core.
+
+That's why Front-Commerce uses a mechanism called **translations fallbacks**.
+Instead of relying on a single file for translations, Front-Commerce will look
+out for translations that are defined by extensions
+
+#### from your extension [declared in `front-commerce.config.js`](/docs/remixed/guides/register-an-extension)
+
+```ts title="./my-extension/index.ts"
+import { defineExtension } from "@front-commerce/core";
+
+export default defineExtension({
+  // ...
+  translations: "./my-extension/translations", // path to your translations
+});
+```
+
+- `<my-extension>/translations/[short-locale].json` or
+- `<my-extension>/translations/[locale].json`
+
+#### from your project
+
+- `lang/[short-locale].json` or
+- `lang/[locale].json`
+
+`[short-locale]` here means that we are only taking the first particle of the
+`locale`. E.g. if the locale was `en-GB`, the short-locale would be `en`. That's
+how `en-GB` would load translations from both `en.json` and `en-GB.json` files.
+
+If a translation key is defined in multiple files, the last one (according to
+the above list) will be be used. This is especially useful if you want to change
+the core's translations.
+
+:::tip ProTip™
+
+You can see exactly which translation files are used by opening the files
+located in `.front-commerce/compiled-lang/`.
+
+:::
+
+Please keep in mind that when you run `pnpm run translate`, new keys will be
+added to `lang/[locale].json` as the script only detects locales in the `app`
+folder. However, if you are developing an extension, you will need to add the it
+can make sense to add the translations to
```

Comment:
**issue(?):** I don't think this sentence makes sense, and I'm not sure what you meant to write 😅